ci-operator/step-registry/cucushift/installer/check/ibmcloud/cucushift-installer-check-ibmcloud-commands.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+
+function run_command() {
+    local CMD="$1"
+    echo "Running Command: ${CMD}"
+    eval "${CMD}"
+}
+
+#OCP-47390 - [IPI-on-IBMCloud] Install cluster with "install-config.yaml" last step
+function checkProviderType {
+    local platformStatusFile="${ARTIFACT_DIR}/platformStatus"
+    oc get infrastructure/cluster -o yaml > ${platformStatusFile}
+    yq-go r ${platformStatusFile} 'status.platformStatus'
+    providerType=$(yq-go r ${platformStatusFile} 'status.platformStatus.ibmcloud.providerType')
+    if [[ ${providerType} != "VPC" ]]; then
+        echo "ERROR: ${providerType} is not expected providerType [VPC]!!"
+        return 1
+    else 
+        return 0
+    fi
+}
+
+#OCP-47395 - [IPI-on-IBMCloud] Install cluster with realtime (RT) kernel
+function checkEnabledRealtimeKernel {
+    local rets nodes i tmp
+    rets=$(oc get machineconfigs | grep realtime-worker)
+    echo ${rets}
+    if [[ ${rets} =~ "realtime-worker" ]] ; then
+        mapfile -t nodes < <(oc get nodes --selector='node-role.kubernetes.io/worker' --no-headers | awk '{print $1}')
+        for i in "${nodes[@]}"; do
+            tmp=$(oc debug node/$i -- uname -a;)
+            echo ${tmp}
+            if [[ ! ${tmp} =~ RT ]]; then
+                echo "ERROR: fail to find RT kennel running !!"
+                return 1
+            fi
+        done
+    else
+        echo "ERROR: Fail to get the expected kerneltype in the machineconfigs"
+        return 1
+    fi
+    return 0
+}
+
+#OCP-47396 - [IPI-on-IBMCloud] Install cluster with FIPS enabled
+function checkEnabledFips {
+    mapfile -t nodes < <(oc get nodes --no-headers | awk '{print $1}')
+    for i in "${nodes[@]}"; do
+        tmp=$(oc debug node/$i -- sysctl crypto.fips_enabled;)
+        echo ${tmp}
+        if [[ ! ${tmp} =~ crypto.fips_enabled\ \=\ 1 ]]; then
+            echo "ERROR: check the fips enabled failed on node ${i}!!"
+            return 1
+        fi
+    done
+    return 0
+}
+
+HasError=0
+
+if [ -f "${SHARED_DIR}/kubeconfig" ] ; then
+    export KUBECONFIG=${SHARED_DIR}/kubeconfig
+else
+    echo "ERROR: fail to get the kubeconfig file under ${SHARED_DIR}!!"
+    exit 1
+fi
+
+if [ -f "${SHARED_DIR}/proxy-conf.sh" ] ; then
+    source "${SHARED_DIR}/proxy-conf.sh"
+fi
+
+checkProviderType
+
+set -x
+set +o errexit
+((HasError+=$?))
+
+if [ -n "${RT_ENABLED}" ] && [ "${RT_ENABLED}" = "true" ]; then
+    checkEnabledRealtimeKernel
+    ((HasError+=$?))
+fi
+
+if [ -n "${FIPS_ENABLED}" ] && [ "${FIPS_ENABLED}" = "true" ]; then
+    checkEnabledFips
```

Comment:
We do not need the check from installer team, there is a dedicate team who is running a full checking for FIPS.

Reply:
so I need to do nothing for [OCP-47396 - [IPI-on-IBMCloud] Install cluster with FIPS enabled](https://polarion.engineering.redhat.com/polarion/redirect/project/OSE/workitem?id=OCP-47396) right? 
fips-check has covered

Reply:
right.

Reply:
checked the existing job, just added FIPS_ENABLED: "true", failed to find call fips-check
ref https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/logs/periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-ibmcloud-ipi-private-fips-f28-destructive/1705335794773790720

> - as: ibmcloud-ipi-private-fips-f14
  cron: 47 17 10,25 * *
  steps:
    cluster_profile: ibmcloud-qe
    env:
      E2E_RUN_TAGS: '@amd64 and @ibmcloud-ipi and @network-ovnkubernetes'
      FIPS_ENABLED: "true"
      TAG_VERSION: '@4.14'
    test:
    - chain: openshift-e2e-test-qe
    workflow: cucushift-installer-rehearse-ibmcloud-ipi-private

Reply:
thanks a lot, I find it on ibmcloud job https://gcsweb-qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/qe-private-deck/logs/periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-ibmcloud-ipi-private-fips-f14/1704414876102823936/artifacts/ibmcloud-ipi-private-fips-f14/fips-check-fips-or-die/